sympy/physics/quantum/tests/test_commutator.py

```diff
@@ -25,9 +26,16 @@ def test_commutator_identities():
     assert Comm(A, B*C).expand(commutator=True) == Comm(A, B)*C + B*Comm(A, C)
     assert Comm(A*B, C*D).expand(commutator=True) == \
         A*C*Comm(B, D) + A*Comm(B, C)*D + C*Comm(A, D)*B + Comm(A, C)*D*B
+    assert Comm(A, B**2).expand(commutator=True) == Comm(A, B)*B + B*Comm(A, B)
+    assert Comm(A**2, C**2).expand(commutator=True) == \
+        Comm(A*B, C*D).expand(commutator=True).replace(B, A).replace(D, C)
```

Comment:
Can this test be changed so this is explicitly listed what is expected on the rhs?

Reply:
Purpose was to show that results from power and multiplication are the same, but expression `A*A` gets automatically replaced with `Pow(A, 2)` so I had to use expand and replace.
I'll add second equality to final expression.

Reply:
You can test that both things equal the same:
```
expected = ...
assert expr1 == expected
assert expr2 == expected
```